less/index.less

```diff
@@ -49,11 +62,237 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+// :root {
+// 	font-size: 62.5%;
+// }
+
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-family: 'Quicksand', sans-serif;
+}
+
+h1, h2, h3 {
+	text-transform: uppercase;
+	text-align: center;
+	font-weight: bold;
+  }
+
+h2 {
+		font-size: @mediumFontSize + .5rem;
+		color: @secondaryColor;
+  }
+
+h3 {
+	font-size: @mediumFontSize + .2rem;
+}
+
+p {
+	line-height: 1.3rem;
+}
+
+.container {
+    display: flex;
+	width: 100%;
+	flex-wrap: wrap;
+	justify-content: center;
+}
+
+.a-mixin() {
+	text-decoration: none;
+	color: @primaryColor;
+
+	&:hover {
+		color: @blackButtonColor;
+	}
+}
+.section-mixin() {
+	width: 90%;
+	margin: 0 3.125rem;
+	padding: 3.125rem;
+	justify-content: center;
+}
+
+nav {
+	width: 100%;
+	font-weight: bold;
+	justify-content: center;
+	ul{
+		display: flex;
+		justify-content: center;
+
+		@media(max-width: @mobileBreakPoint) { 
+			flex-direction: column;
+			align-items: center;
+		}
+	  }
+
+	  li {
+		margin: 3.125rem;
+
+		@media(max-width: @mobileBreakPoint) { 
+			margin: 1rem 0;
+		}
+	  }
+
+	  a {
+		.a-mixin();
+	}
+
 }
 
+  header{
+	  .section-mixin();
+
+	  display: flex;
+	  justify-content: space-around;
+	//   height: 37.5rem;
+	  background-color: @primaryBackgroundColor;
+	  align-items: center;
+
+	  @media(max-width: @tabletBreakPoint) { 
+		flex-direction: column;
+	  }
+
+	  @media(max-width: @mobileBreakPoint) { 
+		flex-direction: column;
+	  }
+	  
+	  .header-image {
+		width: 49%;
+
```

Comment:
This is fine, I don't believe you nested too deeply, but I feel this header section is too long. 
Maybe you could want to break it in 2 or even 3 parts. 
For example you could take out from  `header {...}` this header-image class and header-text class too (because of specificity you don't need to have it inside your header).